activity_duration/src/pulse_points.rs

```diff
@@ -0,0 +1,81 @@
+use std::time::Duration;
+
+use crate::{Activity, ActivityKind};
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct PulseRecord {
+    pub duration: Duration,
+    pub category: PulseRateCategory,
+}
+
+impl From<(PulseRateCategory, Duration)> for PulseRecord {
+    fn from((category, duration): (PulseRateCategory, Duration)) -> Self {
+        Self { duration, category }
+    }
+}
+
+impl From<Activity> for PulseRecord {
+    fn from(Activity { kind, duration, .. }: Activity) -> Self {
+        Self {
+            duration,
+            category: kind.into(),
+        }
+    }
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum PulseRateCategory {
+    Low,
+    Medium,
+    High,
+}
+
+impl From<ActivityKind> for PulseRateCategory {
+    fn from(value: ActivityKind) -> Self {
+        match value {
+            ActivityKind::VO2 | ActivityKind::Anaerobic | ActivityKind::Aerobic => Self::High,
+            ActivityKind::FatBurn | ActivityKind::WarmUp => Self::Medium,
```

Comment:
As I know fat burn is the low intensity aerobic  exercises with long duration. And I suggest that it should be PulseRateCategory::Low.